.circleci/config.yml

```diff
@@ -47,13 +39,11 @@ jobs:
     <<: *default_executor
     steps:
       - checkout
-      - restore_cache:
-          key: *base_config_key
       - restore_cache:
           key: *yarn_cache_key
       - run:
-          name: Install Js dependencies
-          command: yarn install --no-progress --registry https://registry.verdaccio.org
```

Comment:
don't we need this `--registry https://registry.verdaccio.org`?

Reply:
If we use pure-lockfile option, it will use only the uris defined in the yarn.lock.

Reply:
Ok, it's fine, we use `lockfile-lint`